src/db/migrations/20200220144524-create-request.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       autoIncrement: true,
       primaryKey: true,
       type: Sequelize.INTEGER,
-      unique: true,
+      unique: true
```

Comment:
Why are you editing this file?

Reply:
Those are eslint errors that require a trailing comma